### PR TITLE
make map trail color configurable (fix #7660)

### LINKED ISSUE #7660: Make trail more visible
**Is your feature request related to a problem? Please describe.**
Request from support mail:
I am happy, that the trail is now more permanently shown, but the white/light grey color of the trail makes it hardly recognizable between the streets on the map.

**Describe the solution you'd like**
A distinguishable color would be good (most tracking apps I have use blue), or even configurable color.

**Describe alternatives you've considered**
None

**Additional context**

### PR DESCRIPTION
**Pull request by moving-bits**
make map trail color configurable (fix #7660)

map view:
image

settings menu: (Map => trail color)
image

select one of the predefined colors: (grey is default)
image

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1660,4 +1660,9 @@
     <string name="ask_again">Ask me again</string>
     <string name="location_permission_request_explanation">c:geo needs access to the GPS on your device to locate your position and calculate distance and direction to geocaches. It cannot be used without this permission.</string>
     <string name="storage_permission_request_explanation">c:geo will write data onto your phone storage or SD card as soon as you save geocaches for offline use. Furthermore c:geo will use your phone storage for import and export of files and reading of offline maps. It cannot be used without this permission.</string>
+
+    <!-- colors -->
```
Comment (Lineflyer) [inline main/res/values/strings.xml]: AFAIK such single words should also start with a capital letter in english if they are used as a heading or selection list...but I did not look it up.

Comment (moving-bits) [reply main/res/values/strings.xml]: I'm unsure, and found different opinions on the web on this. Some examples:
- Start every sentence with a capital letter? Obvious - but does a single word form a sentence? (eg: https://www.thoughtco.com/guidelines-for-using-capital-letters-1691724)
- Capitalize the first and the last word (eg: https://capitalizemytitle.com/ - New York Times style and Wikipedia style)
- List entities start with _lowercase_ letters if the don't consist of a whole sentence (eg: https://getitwriteonline.com/articles/vertical-lists/).

???

Comment (Lineflyer) [reply main/res/values/strings.xml]: Yes, its difficult. Looking at our app we IMHO tend to capitalize menu entries independently if its a whole sentence or a fragment. Therefore we should just continue this style. And additionally lower case first letter hurts my german language focussed brain ;)

Comment (moving-bits) [reply main/res/values/strings.xml]: ok, capitalization won :-)

**Pull request merged**